firestore/query.go

```diff
@@ -1222,6 +1226,14 @@ type AggregationQuery struct {
 	aggregateQueries []*pb.StructuredAggregationQuery_Aggregation
 	// query contains a reference pointer to the underlying structured query.
 	query *Query
+	//  tx points to an already active transaction within which the AggregationQuery runs
+	tx *Transaction
```

Comment:
can we get this from somewhere else?

Reply:
No. The transaction isn't available in query or client object.

Reply:
Ideally even in Datastore, Transaction method should have been on AggregationQuery object similar to what I have done in this PR. This is because protos allow you to specify read options i.e. a transaction id in [RunAggregationQuery](https://github.com/googleapis/go-genproto/blob/main/googleapis/datastore/v1/datastore.pb.go#L550) request  as well as [RunQueryRequest](https://github.com/googleapis/go-genproto/blob/main/googleapis/datastore/v1/datastore.pb.go#L347)